test/DynamoCoreTests/PythonMigrationAssistantTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Dynamo.PythonMigration.MigrationAssistant;
+using NUnit.Framework;
+using System.Collections.Generic;
+
+namespace Dynamo.Tests
+{
+    public class PythonMigrationAssistantTests
+    {
+        [Test]
+        public void CanMigratePyton2CodeToPython3Code()
+        {
+            // Arrange
+            var expectedPython3Code = @"print(""Hello, Dynamo!"")";
+
+            var python2Code = @"print ""Hello, Dynamo!""";
+
+            var inputNames = new List<string>()
+            {
+                "code"
+            };
```

Comment:
feels weird to have to do this just to run a migration. if it's a required implementation detail, it should be handled in the migration tool and hidden from users.